### ISSUE TITLE
FlatLaf not loading

### ISSUE THREAD

**Issue by wyskoj** [wyskoj/midis2jam2#121]
This is extremely bizarre...

![Screenshot_2022-07-15-23-05-29-902_com miui gallery2](https://user-images.githubusercontent.com/31376393/179370287-73906acd-fea1-4ddf-bb2b-ac4e3e775795.jpg)

**Comment by DevCharly** [comment]
Can imaging one possibility where this could happen: FlatLaf 1.1.2 or older is somewhere on classpath.
Older FlatLaf versions do not have `setup()` methods. They use `install()` methods.

The class `FlatDarkLaf` is probably found because otherwise a `ClassNotFoundException` would be thrown.

You could use

~~~java
FlatDarkLaf.install();
~~~

or

~~~java
try {
    UIManager.setLookAndFeel( new FlatDarkLaf() );
} catch( Exception ex ) {
    System.err.println( "Failed to initialize LaF" );
}
~~~

to avoid this. (with the "risk" that a old FlatLaf version is used if my initial assumption is right)

BTW the "fix" in commit a3abffe99d18cb46889587de7f2500a37cfeac12 probably does not work because `NoSuchMethodError` is not caught because it is not a subclass of `Exception`. Better catch `java.lang.Throwable` or `java.lang.Error`.

**Comment by wyskoj** [comment]
Thank you, this makes sense, but what would cause an older version of FlatLaf to be on the classpath? I package this application with FlatLaf 2.1, so I'm not sure where it would come from...

Also thanks for catching that slip-up :)
